styles/config.less

```diff
@@ -129,3 +114,29 @@ html { font-size: @font-size; }
     .compact-mode();
   }
 }
+
+// Small tabs mode ----------------------------------------------
+
+@theme-tabsizing: ~'theme-@{ui-theme-name}-tabsizing';
+
+[@{theme-tabsizing}="minimum"] {
+    .minimum-width-tabs();
+}
+
```

Comment:
I like @50Wliu suggestion of adding an `auto` option. Then it behaves like it currently does. Minimum in a small window, even in a larger window. Adding a media query like below should work:

```less
@media (max-width: 1280px), (max-height: 800px) {
  [@{theme-tabsizing}="auto"] {
    .minimum-width-tabs();
  }
}
```